src/ECER.Tests/ECER.Tests.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <AssemblyName>ECER.Tests</AssemblyName>
```

Comment:
why is this needed?

Reply:
No longer needed. Initially, I used the [InternalVisible] annotation to grant the Test class access to the new validation engine, but I ultimately decided to make the constructor public instead.